.drone.yml

```diff
@@ -16,11 +16,6 @@ steps:
     depends_on:
       - build project
 
-  - name: sonar scanner
```

Comment:
Out of scope of this change, while I agree it should be removed, I'd rather we bother ACP to give us the developer edition...